From 6118c6ec1291a5c731aa8d4f0c1eb8cce43d47ad Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Mon, 4 Aug 2014 09:04:08 +0200
Subject: [PATCH] Fix out-of-tree builds

The include flag should point to the source directory, not the build
directory.

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
Upstream-status: Submitted [https://github.com/libusbg/libusbg/pull/9]
---
 examples/Makefile.am | 2 +-
 src/Makefile.am      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/Makefile.am b/examples/Makefile.am
index f9f9407..ef4f81e 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -1,5 +1,5 @@
 bin_PROGRAMS = show-gadgets gadget-acm-ecm 
 gadget_acm_ecm_SOURCES = gadget-acm-ecm.c
 show_gadgets_SOURCES = show-gadgets.c
-AM_CPPFLAGS=-I../include/
+AM_CPPFLAGS=-I$(top_srcdir)/include/
 AM_LDFLAGS=-L../src/ -lusbg
diff --git a/src/Makefile.am b/src/Makefile.am
index d955a4c..2f71366 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,4 +1,4 @@
 lib_LTLIBRARIES = libusbg.la
 libusbg_la_SOURCES = usbg.c
 libusbg_la_LDFLAGS = -version-info 0:1:0
-AM_CPPFLAGS=-I../include/
+AM_CPPFLAGS=-I$(top_srcdir)/include/
-- 
1.9.0

